pkg/api/helpers.go

```diff
@@ -154,6 +154,17 @@ func IsStandardQuotaResourceName(str string) bool {
 	return standardQuotaResources.Has(str)
 }
 
+// AddStandardQuotaResourceNames add the resource names into the list of
+// resources known to the quota tracking system
+func AddStandardQuotaResourceNames(names ...ResourceName) {
+	for _, name := range names {
+		if len(strings.Split(string(name), "/")) != 2 {
```

Comment:
@deads2k @derekwaynecarr not sure if we want to do this here, but my idea was that whoever calls this from external/group API, must be complaint with the format rules